README.md

```diff
@@ -1,4 +1,13 @@
-JMEDAS
+JMEValidator
 ======
 
-JME POG CMS Data Analysis School (CMSDAS) exercise
+Common framework for CMS JEC / JER analyzes.
+
+### Dependencies
+
+This package depends on [TreeWrapper](https://github.com/blinkseb/TreeWrapper). Inside the `src` folder of you CMSSW release, do:
+
+```sh
+git clone git@github.com:blinkseb/TreeWrapper.git JMEAnalysis/TreeWrapper
+scram b -j8
+```
```

Comment:
Yep sure.
For the SetBranchStatus, currently, all the branches are disabled by default (status to 0) and only enabled when a read() is requested.